cdap-support-bundle/src/main/java/io/cdap/cdap/support/services/SupportBundleGenerator.java

```diff
@@ -152,6 +171,72 @@ void deleteOldFoldersIfExceedLimit(File baseDirectory) throws IOException {
     }
   }
 
+  /**
+   * Deletes old folders after certain number of folders exist
```

Comment:
This comment is not correct. The method is just doing a recursive directory removal. Also, you can just use `DirUtils.deleteDirectoryContents` instead of reimplementing it.